worlds/stardew_valley/content/vanilla/pelican_town.py

```diff
@@ -229,10 +233,9 @@
             ShopSource(money_price=20000, shop_region=LogicRegion.bookseller_3),),
         Book.mapping_cave_systems: (
             Tag(ItemTag.BOOK, ItemTag.BOOK_POWER),
-            CompoundSource(sources=(
-                GenericSource(regions=(Region.adventurer_guild_bedroom,)),
-                ShopSource(money_price=20000, shop_region=LogicRegion.bookseller_3),
-            ))),
+            GenericSource(regions=(Region.adventurer_guild_bedroom,)),
+            ShopSource(money_price=20000, shop_region=LogicRegion.bookseller_3),
```

Comment:
I would remove the bookseller entirely from this book for reasons discussed in voice chat